pkg/helm/controller/reconcile.go

```diff
@@ -744,10 +743,9 @@ func ValidateNotebook(ctx context.Context, kind string, namespace string) error
 	}
 	if namespace != "redhat-ods-applications"{
 		return errors.New("notebook resource should be created in redhat-ods-applications project to integrate notebook image with the jupyter hub")
-	}else{
-		if  RHODSNotInstalled(ctx) {
-			return errors.New("rhods operator is required to deploy the notebook image integration with the jupyter hub")
-		}
+	}
+	if  DeploymentNotInstalled(ctx, "rhods-operator", "redhat-ods-operator") || DeploymentNotInstalled(ctx, "opendatahub-operator", "openshift-operators") {
+		return errors.New("RHODS operator or ODH operator is required to deploy the notebook image integration with the jupyter hub")
```

Comment:
Is that the correct, full name?
```suggestion
		return errors.New("RHODS operator or ODH operator is required to deploy the notebook image integration with the JupyterHub")
```